libs/s25main/desktops/dskCampaignVictory.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2024 Settlers Freaks (sf-team at siedler25.org)
+//
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "dskCampaignVictory.h"
+#include "Loader.h"
+#include "WindowManager.h"
+#include "desktops/dskMainMenu.h"
+#include "network/GameClient.h"
+
+dskCampaignVictory::dskCampaignVictory(unsigned char chapter)
+    : Desktop(LOADER.GetImageN(ResourceId{chapter ? "setup896" : "setup895"}, 0))
```

Comment:
What is zero? It should never be zero shouldn't it? If it can please add a comment here or at the return below